packages/gitbook-docs/src/markdoc/attributes.ts

```diff
@@ -2,6 +2,15 @@ import { Config } from '@markdoc/markdoc';
 import path from 'path';
 import { ParseContentConfig } from '../parseContent';
 
+const getCorrectGuide = (guide: string) => {
+  switch (guide) {
+    case 'manuale-operativo-dei-servizi':
+      return 'manuale-servizi';
+    default:
+      return guide;
+  }
+};
+
```

Comment:
We need to verify if other guides' name changes between docs and our internal URLs

Reply:
@certevol is going to map space names from docs.pagopa.it to the developer portal

Reply:
Added the link to the file in "DEV-863. "The main difference is the direct link on docs.pagopa to the version chosen as the main one. In some cases, the manual's name on docs.pagopa includes also the product name, which would be redundant in the developer portal's slug. Others can be aligned.